### DISCUSSION TITLE
How to make the effects of LoRa more noticeable?

### DISCUSSION THREAD

**Discussion by fatinghenji** [lllyasviel/Fooocus#848] [Q&A] [answered]
Hi all, I'm having a problem with fooocus: when I want to use lora to do output a specific style of image. lora tends to have minimal effect, and even though I've pulled the weights up to 2.0, the effect is still not noticeable.
I have a few points I would like to get help on:
1. with the current framework, how should I choose the right version of lora? should I go for sd1.5 or sdxl version?
2. how should I optimize the effect of lora? How should I choose the options in the style tab to enhance it?

**Comment by kubilaykilinc** [comment, marked as answer]
Many Lora has trigger words. You have to put that word or words in to end of the Positive Prompt.
and SD 1.5 Lora can't work with Fooocus. (actually doesn't work SD1.5 Lora with SDXL base models)
